### PR TITLE
Srt-12908 Compress edited version of create_server_wrapper

### PR DESCRIPTION
**Pull request by SoneWil**
## What
Add .gitattributes
Update Dockerfile
Add edited version of create_server_wrapper shell script as create_server_wrapper_edited.sh

## Why
Added .gitattributes to prevent git from changing end of line characters and file encoding.
Update Dockerfile to give execute permission on the script directory and to run the edited version of create_server_wrapper.
Added create_server_wrapper_edited.sh to prevent it from being overridden when a newer version of the Hafas deliveries are downloaded.

## Jira
https://trainline.atlassian.net/browse/SRT-12908

### CODE REVIEW

**Comment by pieteretl** [comment]
Hello, what is in the tar? Is it just that file.
Also I have noted that you can edit the file in notepad++ without it encoding/decoding the file incorrectly, however VSCode  incorrectly encodes/decodes...

File: management/script/create_server_wrapper.sh [hunk 1]
```diff
@@ -318,17 +318,17 @@ create_dir ${SERVER_DIR} "the HAFAS server binary"
...
-        | sed "s�^PRIMARY_NAME=\"\"$�PRIMARY_NAME=\"${PRIMARY_NAME}\"�" \
-        | sed "s�^SECONDARY_NAME=\"\"$�SECONDARY_NAME=\"${SECONDARY_NAME}\"�" \
-        | sed "s�^PLAN_VERSION=\"\"$�PLAN_VERSION=\"${PLAN_VERSION}\"�" \
-        | sed "s�^PLAN_SUFFIX=\"\"$�PLAN_SUFFIX=\"${PLAN_SUFFIX}\"�" \
-        | sed "s�^PRODUCT_STATE=\"\"$�PRODUCT_STATE=\"${PRODUCT_STATE}\"�" \
-        | sed "s�^SERVER_TYPE=\"std\"$�SERVER_TYPE=\"${SERVER_TYPE}\"�" \
-        | sed "s�^SERVER_TYPE=\"main\"$�SERVER_TYPE=\"${SERVER_TYPE}\"�" \
-        | sed "s�^PORT=\"\"$�PORT=\"${PORT}\"�" \
-        | sed "s�^CONFIGURED=\"NO\"$�CONFIGURED=\"YES_BUT_NOT_REVIEWED\"�" \
+        | sed "s�^# Provides: primary_name-product_state-hafas-server-secondary_name$�# Provides: ${INT_NAME}�" \
```
[the remark this answers is not in the document]
Comment (SoneWil) [reply management/script/create_server_wrapper.sh]: Reverted all changes to this file

**Comment by SoneWil** [comment]
> Hello, what is in the tar? Is it just that file. Also I have noted that you can edit the file in notepad++ without it encoding/decoding the file incorrectly, however VSCode incorrectly encodes/decodes...

Thank you, yes, VSCode does not support ANSI, but notepad++ does.

**Comment by SoneWil** [comment]
> Hello, what is in the tar? Is it just that file. Also I have noted that you can edit the file in notepad++ without it encoding/decoding the file incorrectly, however VSCode incorrectly encodes/decodes...

The tar file has been removed. It was just that file, but with the .gitattributes the file no longer gets messed up by git :)

**Review by pieteretl — APPROVED** [review]

**Pull request merged**